src/components/Todo/EditionForm.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Datepicker from "react-datepicker";
+
+export default class EditionForm extends React.Component {
+  state = {
+    editTodoName: this.props.todo.name,
+    editDueDate: this.props.todo.dueDate
+   }
+
+  render() {
+    const { onUpdate, todo } = this.props;
+    const { editTodoName, editDueDate } = this.state;
+    const canSubmit = editTodoName === "" || editDueDate === null;
+
+    return (
+      <form
+        onSubmit={e => {
+          e.preventDefault();
+          if (!canSubmit) return;
```

Comment:
This line would be correct if canSubmit was correct, so keep it.

Reply:
excellent news hahaha